### DISCUSSION TITLE
[Solved] PC Engine rename not working from version 6

### DISCUSSION THREAD

**Discussion by mela800** [sanni/cartreader#355] [Need help]
Since version 6, when the checksum matches PCE_CRC_LIST.txt, an error is displayed and the file is not renamed.
The newline code in PCE_CRC_LIST.txt is set correctly.
I do not have the knowledge to do this, so any help would be appreciated.

**Comment by sanni** [comment]
Ok I found the issue, some random lines of code got added to pce.ino when I did change to the new sdfat library.

**Comment by mela800** [comment]
Hi Sanni
Thank you for your prompt reply.
New version worked fine.
We appreciate your help resolving the problem.
